bindings/java/iota-client-java/src/test/java/org/iota/ApiTest.java

```diff
@@ -0,0 +1,61 @@
+package org.iota;
+
+import org.iota.apis.UtilsApi;
+import org.iota.types.*;
+import org.iota.types.ids.OutputId;
+import org.iota.types.ids.TransactionId;
+import org.iota.types.secret.GenerateAddressesOptions;
+import org.iota.types.secret.GenerateBlockOptions;
+import org.iota.types.secret.MnemonicSecretManager;
+import org.junit.jupiter.api.BeforeEach;
+
+public abstract class ApiTest {
+
+    protected static final String DEFAULT_DEVNET_NODE_URL = "https://api.alphanet.iotaledger.net";
+    protected static final String DEFAULT_DEVNET_FAUCET_URL = "https://faucet.alphanet.iotaledger.net";
```

Comment:
Alphanet is a testnet and not a devnet :trollface:

Reply:
Should we rename it to DEFAULT_TESTNET_NODE_URL and DEFAULT_TESTNET_FAUCET_URL ? 
we would need to update in the rust lib too

Reply:
Currently they should also not be added to the lib, there will be new urls when it's public for everyone

Reply:
fixed